### PR TITLE
fix(receive): qr content

### PR DESCRIPTION
**Pull request by banklesss**
##### Ticket

YOMO-1847

### CODE REVIEW

File: apps/wallet-mobile/src/components/ShareQRCodeCard/ShareQRCodeCard.tsx [hunk 1]
```diff
@@ -13,7 +13,8 @@ import {Space} from '../Space/Space'
 import {CaptureShareQRCodeCard} from './CaptureShareQRCodeCard/CaptureShareQRCodeCard'
 
 type ShareQRCodeCardProps = {
-  content: string
+  qrContent: string
+  shareContent: string
```
Comment (michalszorad) [inline apps/wallet-mobile/src/components/ShareQRCodeCard/ShareQRCodeCard.tsx:17]: What is the difference between these two please?

Comment (stackchain) [reply apps/wallet-mobile/src/components/ShareQRCodeCard/ShareQRCodeCard.tsx:17]: I believe shared is carrying all text, while qrContent is just the link, yet needs confirmation from @banklesss

Comment (banklesss) [reply apps/wallet-mobile/src/components/ShareQRCodeCard/ShareQRCodeCard.tsx:17]: Yeah you are right @stackchain. Maybe we can improve the wording. I can take a look at it

**Review by michalszorad — APPROVED** [review]
Maybe just the wording of params qrContent and shareContent, as it was not quite clear from the beginning, but looks good.

**Pull request merged**